### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by yashrajmungale**
Modified the legal notices section as  per the update from CELA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ To help alleviate this pain, we're happy to bring you the List Search app templa
...
+This app template is provided under the [MIT License](https://github.com/OfficeDev/microsoft-teams-apps-requestateam/blob/master/LICENSE) terms.  In addition to these terms, by using this app template you agree to the following:
 
-* You are responsible for complying with applicable privacy and security regulations related to use, collection and handling of any personal data by your app.  This includes complying with all internal privacy and security policies of your organization if your app is developed to be sideloaded internally within your organization.
+-	You are responsible for complying with all applicable privacy and security regulations related to use, collection and handling of any personal data by your app.  This includes complying with all internal privacy and security policies of your organization if your app is developed to be sideloaded internally within your organization.
 
-* Microsoft will have no access to data collected through your app.  Microsoft will not be responsible for any data related incidents or data subject requests.
+-	Where applicable, you may be responsible for data related incidents or data subject requests for data collect through your app.
```
Comment (aosolis) [inline README.md:18]: Grammar: "data collected"

File: README.md [hunk 2]
```diff
@@ -11,15 +11,15 @@ To help alleviate this pain, we're happy to bring you the List Search app templa
 
 ## Legal notice
 
-Please read the license terms applicable to this app template [here](https://github.com/OfficeDev/microsoft-teams-list-search-app/blob/master/LICENSE). In addition to these terms, by using this app template you agree to the following:
+This app template is provided under the [MIT License](https://github.com/OfficeDev/microsoft-teams-apps-requestateam/blob/master/LICENSE) terms.  In addition to these terms, by using this app template you agree to the following:
```
Comment (aosolis) [inline README.md:14]: The link is wrong

File: README.md (hunk 1 above)
Comment (yashrajmungale) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (yashrajmungale) [reply README.md]: Corrected it

**Pull request merged**